### DISCUSSION TITLE
Having Trouble Installing Foundry

### DISCUSSION THREAD

**Discussion by Kolade1024** [Cyfrin/foundry-full-course-cu#1588] [Q&A]
After running this command on my terminal 
curl -L https://foundry.paradigm.xyz | bash

I get this error

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:03:00 --:--:--     0
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to raw.githubusercontent.com:443

**Comment by EngrPips** [comment]
hello @Kolade1024 , installing packages can often be the most difficult part of taking a course so i would advice maybe you look up google and maybe you can find a similar problem as your and hopefully how it was solved

**Comment by AbigirlMuchineripi** [comment]
hello @Kolade1024 , I suggest you to follow the below steps:
1. Try disabling your firewall or antivirus temporarily because it may block curl requests. If this doesn't work, consider,
2. Check Github status that is sometimes GitHub may experience downtime or issues. You can check the status of GitHub services on their status page (https://www.githubstatus.com/) to see if there are any ongoing incidents.
3. You can fetch the resource using a different method like manually downloading the file . or
4. Check your internet connection,also consider using a different internet connection to see if the problem is network specific, or recheck the URL maybe there are errors or the page server is down.

I hope this will help.
